build.gradle

```diff
@@ -70,13 +70,13 @@ task smoke(type: Test) {
 
 checkstyle {
   maxWarnings = 0
-  toolVersion = '8.10.1'
+  toolVersion = '8.14'
   // need to set configDir to rootDir otherwise submodule will use submodule/config/checkstyle
   configDir = new File(rootDir, 'config/checkstyle')
 }
 
 pmd {
-  toolVersion = "6.5.0"
+  toolVersion = "6.9.0"
   ignoreFailures = true
```

Comment:
ignoreFailures = true?
What's the point in having it if it doesn't fail

Reply:
For template the flag is irrelevant aka failures can be **not** ignored. For projects using the template the rules can easily blow up out of proportion and it is up to the team/developer to decide on the rules and fix them individually

Reply:
i would say the default should be don't ignore errors, @rkondratowicz has seen what happened on bulk scanning because the default was to ignore them

Teams can decide to change it, but imo you shouldn't ignore by default

Reply:
We are or ought to be aware. Let's put up the vote. I do remember the decision was made to ignore them but can't recall the time and situation